prow/cmd/phaino/local.go

```diff
@@ -314,28 +328,51 @@ func convertJob(ctx context.Context, log *logrus.Entry, pj prowapi.ProwJob, priv
 		return nil
 	}
 	log.Info("Starting job...")
+	// TODO(fejta): default grace and timeout to the job's decoration_config
```

Comment:
Came here to say this

Reply:
Yep this is a good idea and on the agenda.